lib/action_cable/channel/naming.rb

```diff
@@ -0,0 +1,23 @@
+module ActionCable
+  module Channel
+    module Naming
+      extend ActiveSupport::Concern
+
+      class_methods do
+        # Returns the last part of the channels's name, underscored, without the ending
+        # <tt>Channel</tt>. For instance, ChatChannel returns <tt>chat</tt>.
+        #
+        # ==== Returns
+        # * <tt>string</tt>
```

Comment:
Let's make the example more real instead of this returns thing. Like so:

```ruby
# Returns the last part of the channels's name, underscored, without the <tt>Channel</tt> ending.
#
#   ChatChannel.channel_name # => 'chat'
```